apps/web/components/ActionButtons.tsx

```diff
@@ -56,6 +61,20 @@ export function ActionButtons(props: {
     setIsTrashing(false);
   }, [threadId, refetch]);
 
+  const [isReadUnreading, setisReadUnreading] = useState(false);
```

Comment:
`setisReadUnreading` => `setIs`

Reply:
`isReadUnreading` is bad in any case.

`markingRead`, and `setMarkingRead` is better. Or:
`isMarkingRead`, and `setIsMarkingRead` is better.